student.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+# Students' information/actions in the library.
+class Student < Person
+  def initialize(age, classroom, name = 'Unknown')
```

Comment:
- [ ] [REQUIRED] Good job so far! In line 4, you should add `parent_permission = true` after `name = 'Unknown'`!

#

**[HINT] It would like this ``def initialize(age, classroom, name = "Unknown", parent_permission = true)``.**